### ISSUE TITLE
Calling prototype functions crash script w/o @grant none

### ISSUE THREAD

**Issue by Bl4Cc4t** [Tampermonkey/tampermonkey#1385]
## Expected Behavior
Declaring and calling a prototype function should not result in the script crashing.

## Actual Behavior
Upon calling a prototyped function, the script crashes. This does not happen when a `@grant none` is in the header.
The error reads:
```
Uncaught (in promise) TypeError: "".testFunc is not a function
```

Latest Chrome (v96.0.4664.93) and the stable version of TM are both unaffected.

## Specifications

- Firefox v95.0
- TM v4.14.6149 BETA
- Windows 10

## Script

```js
// ==UserScript==
// @name          test #1
// @version       0.1
// @description   prototype test
// @author        Bl4Cc4t
// @include       *
// @grant         GM_getValue
// ==/UserScript==

(function() {
  String.prototype.testFunc = function() {
    console.log("this is a string test!")
  }

  Number.prototype.testFunc = function() {
    console.log("this is a number test!")
  }

  Array.prototype.testFunc = function() {
    console.log("this is an array test!")
  }


  "".testFunc();
  (123).testFunc();
  [1, 2, 3].testFunc();
})()
```

**Comment by derjanb** [comment]
Should be fixed at 4.14.6150 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_4_14_6150.crx))

**Comment by Bl4Cc4t** [comment]
just tested it again, it works now. thanks!
